### PR TITLE
ci: add asf license check

### PR DESCRIPTION
**Pull request by QuakeWang**
This PR will add [skywalking-eyes](https://github.com/apache/skywalking-eyes) for checking ASF license header.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,8 +1,16 @@
-#[cfg(test)]
```
Comment (Xuanwo) [inline src/lib.rs:1]: Remove this block could make CI to fail. We can remove them after we have added real code here.

**Review by Xuanwo — COMMENTED** [review]
Mostly LGTM, thanks a lot!

File: src/lib.rs (hunk 1 above)
Comment (QuakeWang) [reply src/lib.rs:1]: > Remove this block could make CI to fail. We can remove them after we have added real code here.

OK, I have updated it, PTAL.

**Comment by Xuanwo** [comment]
Hi, @JingsongLi, I believe this PR is ready for merging.

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**